index.html

```diff
@@ -15,16 +15,171 @@
+                        <p class="css">css</p>
+                        <p class="javascript">javascript</p>
+                    </div>
+                    <button type="button" class="see-prjct">See Project</button>
+                </div>
+               
+            </div>
+
+            <!-- CARD 2 (Multi-post stories)-->
+            <div class="works-card1">
+                <div class="child-worksCard-info">
+                    <img src="./images/SnapshootCard2.png" alt="snapshoot">
+                    <h1 class="prjct-title multi-post">Multi-Post <br>Stories</h1>
+                    <div class="prjct-info">
+                        <h6 class="canopy client-name">CANOPY</h6>
+                        <div class="dot-separator"></div>
+                        <h6 class="client-name">Back End Dev</h6>
+                        <div class="dot-separator"></div>
+                        <h6 class="client-name">2015</h6>
+                    </div>
+                    <p class="description">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.</p>
+                    <div class="pro-languages">
+                        <p class="html">html</p>
+                        <p class="css">css</p>
+                        <p class="javascript">javascript</p>
+                    </div>
+                    <button type="button" class="see-prjct">See Project</button>
+                </div>
+               
+            </div>
+
+            <!-- CARD 3 (Tonic)-->
+            <div class="works-card1">
+                <div class="child-worksCard-info">
+                    <img src="./images/SnapshootCard3.png" alt="snapshoot">
+                    <h1 class="prjct-title">TONIC</h1>
+                    <div class="prjct-info">
+                        <h6 class="canopy client-name">CANOPY</h6>
+                        <div class="dot-separator"></div>
+                        <h6 class="client-name">Back End Dev</h6>
+                        <div class="dot-separator"></div>
+                        <h6 class="client-name">2015</h6>
+                    </div>
+                    <p class="description">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.</p>
+                    <div class="pro-languages">
+                        <p class="html">html</p>
+                        <p class="css">css</p>
+                        <p class="javascript">javascript</p>
+                    </div>
+                    <button type="button" class="see-prjct">See Project</button>
+                </div>
+               
+            </div>
+
+            <!-- CARD 4 (Multi-post stories)-->
+            <div class="works-card1">
+                <div class="child-worksCard-info">
+                    <img src="./images/SnapshootCard4.png" alt="snapshoot">
+                    <h1 class="prjct-title multi-post">Multi-Post <br>Stories</h1>
```

Comment:
- [ ] Kindly change this `h1` tag to an `h2` tag. H1 tags are reserved to be used as a title to the website and should be used only once per page.